packages/build-lz-options/src/components/outputOptions.tsx

```diff
@@ -15,8 +15,14 @@ export const OutputOptions: React.FC<OptionOutputProps> = ({
 }: OptionOutputProps) => {
   return (
     <Box flexDirection="column">
+      {props.warning.length > 0 && (
+        <Text color={"yellow"}>
+          <Text bold={true}>Warning:</Text> {props.warning}
+        </Text>
+      )}
       <Text>
-        Result: <Text color={"green"}>{props.hex}</Text>
+        <Text bold={true}>Result:</Text>{" "}
+        <Text color={"green"}>{props.hex}</Text>
```

Comment:
Truly not a functional comment, just FYI code-style:

Truthy boolean attributes don't require a value in React:

```jsx
<Text bold>Bold text</Text>
```

And string attributes don't require `{}`:

```jsx
<Text color="green">Green text</Text>
```